start.bash

```diff
@@ -18,4 +18,14 @@ echo "Disabling chrony on balena host:"
 echo "Config file content:"
 cat "$CONFIG_FILE"
 
-chronyd -d -s -f "$CONFIG_FILE"
+: "${TIME_SYNC_DAEMON:=chronyd}"  # Default to chronyd if unset or empty
+
+if [ "$TIME_SYNC_DAEMON" = "chronyd" ];
+then
+    chronyd -d -s -f "$CONFIG_FILE"
+elif [ "$TIME_SYNC_DAEMON" = "timemaster" ];
+then
+    timemaster -f "$CONFIG_FILE"
```

Comment:
Does this have anything to do with chrony now?

Reply:
> the `timemaster` daemon. This uses ptp4l and phc2sys in combination with chronyd

Yes, timemaster is used to run chronyd.